lib/src/migrators/module.dart

```diff
@@ -0,0 +1,302 @@
+// Copyright 2018 Google LLC
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'dart:io';
+
+// The sass package's API is not necessarily stable. It is being imported with
+// the Sass team's explicit knowledge and approval. See
+// https://github.com/sass/dart-sass/issues/236.
+import 'package:sass/src/ast/sass.dart';
+
+import 'package:args/args.dart';
+import 'package:path/path.dart' as p;
+
+import 'package:sass_migrator/src/migrator_base.dart';
+import 'package:sass_migrator/src/patch.dart';
+import 'package:sass_migrator/src/utils.dart';
+
+import 'module/built_in_functions.dart';
+import 'module/local_scope.dart';
+import 'module/stylesheet_migration.dart';
+
+class ModuleMigrator extends MigratorBase {
+  final name = "module";
+  final description = "Migrates stylesheets to the new module system.";
+  final argParser = ArgParser()
+    ..addFlag('migrate-deps',
+        abbr: 'd', help: 'Migrate dependencies in addition to entrypoints.');
+
+  bool get _migrateDependencies => argResults['migrate-deps'];
```

Comment:
It's probably a good idea to explicitly add `as bool` here. Otherwise, you're relying on an implicit cast, which is likely to become an analysis error at some point.

Also, it's up to you, but I generally prefer to just refer to arguments directly rather than creating indirection through a getter.